pkg/test/integration/remote.go

```diff
@@ -37,13 +38,14 @@ func downloadPath(url, path string) (string, error) {
 	if _, err = hasher.Write([]byte(url)); err != nil {
 		return "", err
 	}
-	dst := filepath.Join(path, string(hasher.Sum32()))
+	dst := filepath.Join(path, fmt.Sprint(hasher.Sum32()))
```

Comment:
```suggestion
	dst := filepath.Join(path, fmt.Sprintf("%x", hasher.Sum32()))
```

I usually like to represent hashes as hexadecimal strings. That said, the only advantage of doing so is probably a shorter string in some cases, so feel free to ignore this suggestion.